src/vs/code/electron-main/window.ts

```diff
@@ -373,6 +373,19 @@ export class VSCodeWindow {
 			}
 		});
 
+		// Swipe command support (macOS)
+		this._win.on('swipe', (e, cmd) => {
```

Comment:
@bpasero Read my mind. I thought about it but I wasn't sure you guys would want it. We could take an object of events and left/right options to compare to. 

Is that OK with you?